lib/interactive_viewer.ml

```diff
@@ -9,7 +9,16 @@ let current_patch patches =
   let$ index = index in
   List.nth_opt patches index
 
-let string_of_operation = Format.asprintf "%a" (Patch.pp_operation ~git:false)
+let string_of_operation operation =
+  match operation with
+  | Patch.Create path -> "Creation of `" ^ path ^ "`"
```

Comment:
The backquotes `` ` .. ` `` don't look very nice, we could use some colors or maybe bold ?

This function is called below like this:
```ocaml
  W.string @@ string_of_operation p.Patch.operation
```
`W.string` has a `~attr` optional argument that can be used to put some text in colors/bold/etc..
See the doc for `W.string`: https://ocaml.org/p/nottui/latest/doc/Nottui_widgets/index.html#val-string

We can then use `Ui.hcat` ([doc](https://ocaml.org/p/nottui/latest/doc/Nottui/Ui/index.html#val-hcat)) to concatenate a list of elements created with `W.string`.